cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,37 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import ArticlePageObject from '../support/pages/article.pageObject';
+
+const articlePage = new ArticlePageObject();
+
 describe('Article', () => {
-  before(() => {
+  let article;
 
+  before(() => {
+    cy.task('db:clear');
```

Comment:
cy.task('db:clear') - why do we need to clear db in before and beforeEach at the same time?